crates/bevy_pbr/src/prepass/prepass_bindings.rs

```diff
@@ -1,82 +1,61 @@
 use bevy_core_pipeline::prepass::ViewPrepassTextures;
 use bevy_render::render_resource::{
-    BindGroupLayoutEntry, BindingType, ShaderStages, TextureAspect, TextureSampleType, TextureView,
-    TextureViewDescriptor, TextureViewDimension,
+    binding_types::{
+        texture_2d, texture_2d_multisampled, texture_2d_u32, texture_depth_2d,
+        texture_depth_2d_multisampled,
+    },
+    BindGroupLayoutEntryBuilder, TextureAspect, TextureSampleType, TextureView,
+    TextureViewDescriptor,
 };
 use bevy_utils::default;
 use smallvec::SmallVec;
 
 use crate::MeshPipelineViewLayoutKey;
 
 pub fn get_bind_group_layout_entries(
```

Comment:
so here, we are not specifying the stage for these bindings, so we rely on the default stage being what we want at the call site. 

probably not an issue but maybe we ought to specify explicitly? it would add quite a lot of noise since it would also need to be done for all the delegated mesh_view_bindings as well.

Reply:
Yeah, I wasn't entirely sure what to do here. Part of the goal of this api was to have the concept of a default visibility and I kinda like that it gives the control to the caller. It's very implicit though. Technically, it doesn't really matter because it's only intended to be used by the mesh_view_bindings anyway.

Reply:
> it's only intended to be used by the mesh_view_bindings anyway.

for now, yes. i plan to make things (prepass outputs in particular) easily available to other pipelines as well though. but then in that case it's probably better that the caller decides the visibility anyway.